src/compiler/types.ts

```diff
@@ -1725,6 +1725,7 @@ namespace ts {
         getSymbolAtLocation(node: Node): Symbol;
         getSymbolsOfParameterPropertyDeclaration(parameter: ParameterDeclaration, parameterName: string): Symbol[];
         getShorthandAssignmentValueSymbol(location: Node): Symbol;
+        getExportSpecifierLocalTargetSymbol(location: ExportSpecifier): Symbol;

```

Comment:
I don't think "target" clarifies that any better, but I'm not familiar enough with where else we make this distinction. I just mention this because we're making this a part of our API, so we'll have to commit to this naming scheme.